features/common.feature

```diff
@@ -42,7 +42,7 @@ Feature: Show common information for tooling
           "inputHint": "SingleLine"
         },
     """
-    When I run `product-type-update --projectKey product-type-json-generator-tests --source product-type-boo-txt-num.json`
+    When I run `product-type-update --projectKey product-type-json-generator-int-test --source product-type-boo-txt-num.json`
```

Comment:
This project had been deleted so I created a new one and updated travis secrets.